lib/dpl/provider/pypi.rb

```diff
@@ -64,7 +65,9 @@ def needs_key?
 
       def push_app
         context.shell "python setup.py register -r #{options[:server] || 'pypi'}"
```

Comment:
I think that implies that the docs need to be updated that people have to register their project manually on the PyPI website to be safe?